prettier.config.js

```diff
@@ -1,18 +0,0 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-module.exports = {
```

Comment:
As discussed offline, thanks to this file local editors are able to pick up the prettier config and auto format the code as you go or on save. Or is this no longer a concern?

Reply:
This config only has 2 rules, quotes and commas. Not super useful.

The standard is to use `gts fix`, which can be triggered as you save if the developer really wishes.

More at:
https://github.com/google/gts